README.rst

```diff
@@ -78,3 +78,24 @@ This will generate a new migration script. Then run:
 To apply the migration.
 
 For a full migration command reference, run ``python manage.py db --help``.
+
+
+Docs
+----
+
+To build the docs:
+
+move to the `slate` directory:
+::
+
+    cd slate/
+
+then build the docs with middleman:
+::
+
+    bundle exec middleman build
```

Comment:
No, that's the exact setup I had. As long as you ran `bundle install` before running the build command